docs/likelihood.rst

```diff
@@ -173,6 +173,8 @@ for a 2-bin channel is shown below:
 
    { "name": "mod_name", "type": "histosys", "data": {"hi_data": [20,15], "lo_data": [10, 10]} }
 
+The previous example will specify the expected event rate for the up-variation of the ``histosys`` as ``[20, 15]`` (20 events in first bin, 15 events in second bin); for the low-variation as ``[10, 10]`` (10 events in first bin, 10 events in second bin). This variation is absolute (not relative!).
```

Comment:
```suggestion
This example specifies the expected event rate for the up-variation of the ``histosys`` as ``[20, 15]`` (20 events in first bin, 15 events in second bin); for the low-variation as ``[10, 10]`` (10 events in first bin, 10 events in second bin).
This variation is absolute (not relative!).
```

Reply:
I'd suggest:
* Talking about "this" example instead of the "previous" as all this text is in the "Correlated Shape (histosys)" section.
* Splitting sentences across multiple lines to make future revisions more clean.